packages/storybook/package.json

```diff
@@ -6,6 +6,7 @@
   "scripts": {
     "build": "storybook build",
     "start:storybook": "storybook dev -p 6006",
+    "test:storybook": "test-storybook",
```

Comment:
Maybe just call the command `test`? That way the CI should automatically pick it up and run it